tests/test_duckdb_verify.py

```
@@ -26,14 +26,12 @@ def test_verify_post_load_fails_on_empty(tmp_path):
     con = duckdb.connect(str(db))
     con.execute("create schema if not exists raw;")
     con.execute(
-        "create table raw.edgar_master (cik text, company_name text, form_type text, date_filed text, filename text)"
+        "create table raw.edgar_master (cik text, company_name text, form_type text, date_filed text, filename text, loaded_at timestamp, index_file_date text)"
     )
     con.close()
 
     try:
         verify_duckdb_post_load()
         assert False, "Expected AssertionError"
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)